common/components/componentsBySection/FindProjects/Filters/RenderFilterCategory.jsx

```diff
@@ -105,9 +105,9 @@ class RenderFilterCategory<T> extends React.PureComponent<Props<T>, State> {
   _renderFilterList(data) {
     //this function renders individual clickable filter items regardless of category or subcategory status
     let sortedTags = Object.values(data).map((tag) =>
-      <li key={tag.category + '-' + tag.display_name} className="ProjectFilterContainer-list-item">
-          <input type="checkbox" id={tag.category + '-' + tag.display_name} checked={this.props.checkEnabled(tag)} onChange={() => this.props.selectOption(tag)}></input>
-          <label htmlFor={tag.category + '-' + tag.display_name}>
+      <li key={tag.category + '-' + tag.tag_name} className="ProjectFilterContainer-list-item">
```

Comment:
```suggestion
      const key:string = tag.category + '-' + tag.tag_name;
      return  (<li key={key} className="ProjectFilterContainer-list-item">
```

So as to avoid the need to copy-paste any changes in the future.

Reply:
Had to do it slightly differently than this exact suggestion but the key is now a const and reused. 👍